src/App.tsx

```diff
@@ -64,7 +65,13 @@ const App: React.FunctionComponent = () => {
         />
         <Route path="/:id/media" element={<ProfilePage tabKey="media" />} />
         <Route path="/:id/likes" element={<ProfilePage tabKey="likes" />} />
-        <Route path="/:id" element={<ProfilePage tabKey="tweets" />} />
+        <Route path="/more" element={<MorePage />} />
+        <Route path="/authorization" element={<AuthorizationPage />} />
+        <Route path="/followers" element={<Followers />} />
+        <Route path="/following" element={<Following />} />
+        <Route path="/trends" element={<Trends />} />
+        <Route path="/tweet/:id" element={<Tweet />} />
```

Comment:
bad merge You had this part two times. Check all your files!!!

Reply:
fix